ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-commands.sh

```diff
@@ -34,7 +32,7 @@ else
     exit 1
 fi
 
-# Lets  check the syntax of yaml file by reading it.
+# Check the syntax of yaml file by reading it.
```

Comment:
again why?  pr changes should really be only about what is necessary not what we think it should be based on word preference. This is absolutely superfluous